### PR TITLE
layerscape:add ls1021a platform support

### PR DESCRIPTION
**Pull request by keinYang**
add armv7 as a new subtarget on layerscape platform, and add ls1021aiot
device in it.

Signed-off-by: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>

### CODE REVIEW

**Comment by yangbolu1991** [comment]
@keinYang 
Good work!
Regarding to the new files copy right, shouldn't it be "Copyright 2017 NXP" ?

**Comment by keinYang** [comment]
I have changed the new files copyright, thanks @yangbolu1991

File: target/linux/layerscape/armv7/config-4.9 [hunk 1]
```diff
@@ -0,0 +1,1462 @@
...
+CONFIG_I2C_DESIGNWARE_CORE=y
+CONFIG_I2C_DESIGNWARE_PLATFORM=y
+CONFIG_I2C_HELPER_AUTO=y
+CONFIG_I2C_IMX=y
+CONFIG_I2C_MUX=y
+CONFIG_I2C_MUX_PCA954x=y
+CONFIG_I2C_MUX_PINCTRL=y
+CONFIG_I2C_NOMADIK=y
+CONFIG_I2C_OMAP=y
+CONFIG_I2C_RK3X=y
```
Comment (hauke) [inline target/linux/layerscape/armv7/config-4.9]: Please clean up this configuration file and only add the drivers which are on the SoC.
I assume you do not have a Rockchip and a TI I2C controller in the SOC.

If it is a driver for an optional component it should be added with a kernel package as a kernel model.

File: target/linux/layerscape/armv7/config-4.9 [hunk 2]
```diff
@@ -0,0 +1,1462 @@
...
+CONFIG_IPV6_ROUTER_PREF=y
+CONFIG_IPV6_SIT=y
+# CONFIG_IPV6_SUBTREES is not set
+# CONFIG_IP_ADVANCED_ROUTER is not set
+# CONFIG_IP_MROUTE is not set
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_IPTABLES=y
+CONFIG_IP_NF_MANGLE=y
+CONFIG_IP_NF_NAT=y
+CONFIG_IP_NF_TARGET_MASQUERADE=y
```
Comment (hauke) [inline target/linux/layerscape/armv7/config-4.9]: Netfilter should be a module

File: target/linux/layerscape/armv7/config-4.9 [hunk 3]
```diff
@@ -0,0 +1,1462 @@
...
+CONFIG_MFD_PALMAS=y
+CONFIG_MFD_RK808=y
+CONFIG_MFD_SEC_CORE=y
+CONFIG_MFD_STMPE=y
+CONFIG_MFD_SYSCON=y
+CONFIG_MFD_TPS65090=y
+CONFIG_MFD_TPS65217=y
+CONFIG_MFD_TPS65218=y
+CONFIG_MFD_TPS6586X=y
+CONFIG_MFD_TPS65910=y
```
Comment (hauke) [inline target/linux/layerscape/armv7/config-4.9]: Please also clean up the MFD drivers, 
These are just examples there is a lot more stuff in there which is not needed.

File: target/linux/layerscape/armv7/config-4.9 (hunk 1 above)
Comment (keinYang) [reply target/linux/layerscape/armv7/config-4.9]: Thanks for your review, I have clean up the configuration file, and update the patch, please review it, thank you.

**Comment by keinYang** [comment]
I have clean up the configuration file "target/linux/layerscape/armv7/config-4.9" , and update the patch

**Comment by blogic** [comment]
With the remerge in progress, all PRs on the lede-project organisation will be closed. Please help getting this merged or rebase/post it on the openwrt project page (https://github.com/openwrt/openwrt/pulls). All remaining PRs will be closed in 30 days.

**Pull request closed without merging**